projects/fmcjesdadc1/zc706/Makefile

```diff
@@ -14,7 +14,6 @@ M_DEPS += ../../../library/jesd204/scripts/jesd204.tcl
 M_DEPS += ../../../library/common/ad_sysref_gen.v
 M_DEPS += ../../../library/common/ad_iobuf.v
 
-LIB_DEPS += axi_ad9250
 LIB_DEPS += axi_clkgen
```

Comment:
Makefile must be updated to reflect the new generic TPL  core used. 
LIB_DEPS += jesd204/ad_ip_jesd204_tpl_adc